beacon-chain/core/validators/validator.go

```diff
@@ -254,15 +254,29 @@ func ValidatorSlotAndRole(pubKey []byte, validators []*pb.ValidatorRecord, shard
 	return 0, pbrpc.ValidatorRole_UNKNOWN, fmt.Errorf("can't find slot number for validator with public key %#x", pubKey)
 }
 
+// TotalBalance returns the total deposited amount in Gwei of all active validators.
```

Comment:
This returns the total effective balance. It seems that validators can over-deposit funds, so this comment is misleading to say the "total deposited amount" when we are only counting the amount "at stake".

Reply:
Very good point. Done